package P_20211223.exception3;

public class Exception03P355 {
	public static void main(String[] args) {
		int [] num = {1,2,3,4,5,0};
		try {
			for(int i = 0; i<6;i++) {
				System.out.println(n);
			}
			System.out.println(num[4]/num[5]);
		}catch (Exception e){
			
		}
	}
}
